src/extension.ts

```diff
@@ -9,9 +9,11 @@ export function activate(context: vscode.ExtensionContext) {
   const codeLensProvider = new JestRunnerCodeLensProvider();
   const config = new JestRunnerConfig();
 
-  const runJest = vscode.commands.registerCommand('extension.runJest', async (argument: object | string) => {
+  const runJest = vscode.commands.registerCommand('extension.runJest', async (argument: object | string | {path: string}) => {
     if (typeof argument === 'string') {
       jestRunner.runCurrentTest(argument);
+    } else if('path' in argument) {
```

Comment:
Just noticed that this will prevent `runCurrentTest` from being called when this is run from the editor context menu and thus always run the entire file instead of only the test where the cursor is.

Think is better to define new commands for the run and debug from the explorer context.